#rosparam load 파일이름
core:
  control:
    is_simulation : true
    is_morai : true
    speed_slow_obejct : 10.0
    gain_err_y : 20.0
    gain_steer : 25.0
    gain_look_ahead_distance : 0.3
    offset_look_ahead_distance : 3.0
    min_look_ahead_distance : 4.0
    max_look_ahead_distance : 25.0
    offset_yaw : 0.0